### PR TITLE
clean up UI and animate live recording circle

### PR DESCRIPTION
**Pull request by dereckquock**
- animate LIVE recording circle
- fix post with no image
- move shared components to `/shared`

## LIVE

gif

### CODE REVIEW

File: components/LiveCard.js [hunk 1]
```diff
@@ -1,44 +1,102 @@
...
+  clockRunning,
+  cond,
+  set,
+  spring,
+  startClock,
+  stopClock,
+  Value,
+} = Animated;
+
+function runSpring(clock, value, dest) {
```
Comment (dereckquock) [inline components/LiveCard.js]: spring animate the `transform: scale`

File: screens/Activate.js [hunk 2]
```diff
@@ -38,7 +38,12 @@ const ActivateScreen = () => {
             WebBrowser.openBrowserAsync(
               'https://docs.google.com/forms/d/e/1FAIpQLSdZ1EBI_kCqt8xtK1n1PBfcBUlHFPl45o-9Ls3O2srwejpjGw/viewform?vc=0&c=0&w=1',
               { toolbarColor: Colors.darkestGray }
-            );
+            ).catch(err => {
```
Comment (dereckquock) [inline screens/Activate.js:41]: handle [this issue](https://github.com/expo/expo/issues/6918)
i'm assuming it'll be fixed in the next expo release

**Review by Phillip3 — CHANGES_REQUESTED** [review]
🚫  Block until the app is approved by Apple 🚫 

Approved! ✅ by 🍎 , so let's get this thing reviewed!

File: components/LiveCard.js (hunk 1 above)
Comment (Phillip3) [reply components/LiveCard.js]: @dereckquock I wanted a soft blink, like fade in/out blinking (like a recording light) not a bounce 😄

Comment (dereckquock) [reply components/LiveCard.js]: i kinda like the bounce 😅 maybe we can work a compromise...lemme see

File: components/LiveCard.js [hunk 3]
```diff
@@ -1,44 +1,102 @@
...
+    state.position,
+  ]);
+}
+
+function AnimatedCircle() {
+  const clock = new Clock();
+  const scale = runSpring(clock, 1, 1);
+
+  return (
+    <Animated.View style={{ transform: [{ scale }] }}>
```
Comment (Phillip3) [inline components/LiveCard.js]:
```suggestion
  const opacity = runSpring(clock, 1, 1);

  return (
    <Animated.View style={{ transform: [{ opacity }] }}>
```

File: components/LiveCard.js [hunk 4]
```diff
@@ -1,44 +1,102 @@
...
 const styles = StyleSheet.create({
+  container: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'center',
+    backgroundColor: Colors.darkestGray,
+  },
   liveText: {
-    ...TextStyles.subtitle,
+    marginRight: 16,
```
Comment (Phillip3) [inline components/LiveCard.js]: Why don't we want to use the TextStyles thing?

**Review by Phillip3 — CHANGES_REQUESTED** [review]

File: components/LiveCard.js (hunk 4 above)
Comment (dereckquock) [reply components/LiveCard.js]: cause we have a `<Text/>` component for handling text 😅 
we should really use that `<Text/>` component for text stuff. and if we want to have a defined type system, then we can build something like a `<Header>` or something  🤔 

we should add props to the `<Text>` component for defined sizing probably

File: components/LiveCard.js (hunk 3 above)
Comment (dereckquock) [reply components/LiveCard.js]: ooh lemme try

File: components/LiveCard.js (hunk 4 above)
Comment (Phillip3) [reply components/LiveCard.js]: Like we talked about, I kind of like the idea of wrapping all of these into
```js
<TitleText/>
<SubtitleText/>
<BodyText/>
```
or something of the sort. Maybe a "later" thing, but I like it.

Comment (dereckquock) [reply components/LiveCard.js]: yah! let's put that as a TODO

File: components/LiveCard.js [hunk 5]
```diff
@@ -1,44 +1,71 @@
...
       </Svg>
-      <View style={{ flex: 1 }} />
+    </Animated.View>
+  );
+}
+
+export default ({ style }) => {
+  return (
+    <View style={[style, styles.container]}>
+      <Text L style={[TextStyles.title, styles.liveText]}>
```
Comment (dereckquock) [inline components/LiveCard.js:47]: here's how you can render large text (`props.L`) and compose styles

File: components/LiveCard.js [hunk 6]
```diff
@@ -1,44 +1,71 @@
...
+import TextStyles from '../constants/TextStyles';
+import Text from './shared/Text';
+
+const { block, set, useCode, Value } = Animated;
+
+function AnimatedCircle() {
+  const animation = new Value(0);
+
+  // loop the opacity
+  // https://www.youtube.com/watch?v=1Q9efh7OcR8
```
Comment (dereckquock) [inline components/LiveCard.js:16]: this guy is amazing! he does the `can it be done in react-native` videos

File: components/shared/Text.js [hunk 7]
```diff
@@ -15,12 +17,27 @@ export default props => {
     fontWeight = 'Bold';
   }
 
+  if (props.XL) {
```
Comment (dereckquock) [inline components/shared/Text.js:20]: can define the size of text by passing in a prop. maybe we should handle this differently later, but this should be cool for now 😎

**Review by Phillip3 — APPROVED** [review]
Woo! Looks good.

**Pull request merged**